utils/vim/Makefile

```diff
@@ -8,19 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vim
-PKG_VERSION:=9.0
+PKG_VERSION:=9.1.0888
 PKG_RELEASE:=1
-VIMVER:=90
+VIMVER:=91
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://ftp.vim.org/pub/vim/unix
-PKG_HASH:=a6456bc154999d83d0c20d968ac7ba6e7df0d02f3cb6427fb248660bacfb336e
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
Please dont do this. We can not download tarball named like this.

Reply:
https://github.com/openwrt/openwrt/blob/main/scripts/download.pl
No, I cant deal with it, why `PKG_SOURCE:=foobar` `PKG_SOURCE_URL:=https://foobar` will download tarball from `https://foobar/foobar`, whyyyyy...

Reply:
Resolved for now?
[download.pl](https://github.com/openwrt/openwrt/blob/main/scripts/download.pl) script behaves different from what it said in help messages

```perl
@ARGV > 2 or die "Syntax: $0 <target dir> <filename> <hash> <url filename> [<mirror> ...]\n";